api/models/task.py

```diff
@@ -18,7 +17,8 @@ class Task(db.Model):
         UUIDType(binary=True), ForeignKey("current_session.user_id"), nullable=False
     )
     title = Column(String, nullable=False)
-    description = Column(String, nullable=False)
+    description = Column(String, nullable=True)
```

Comment:
What's the intuition behind making this null-able? I get it for the `completed` row, because it being null has a distinct meaning (ie. it hasn't been completed yet), but what does it mean for the description being `null`, versus it having a value of `""`?